Gemfile

```diff
@@ -37,7 +37,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
 end
-
+gem 'active_model_serializers', '~> 0.10.12'
```

Comment:
Good choice, but how are you using those serializers in the controllers?

Reply:
Okkk, i included serializers in controllers, user "each_serializer: ExampleSerializer". When tried to do just "serializer: ..." it didnt work, showed me 'undefined method' :)

Reply:
used*

Reply:
With single object we use serializer: with arrays we use each_serializer: